src/lua/luaTask.c

```diff
@@ -227,9 +275,16 @@ static void lua_task(void *params)
         };
 
         int consecutive_failures = 0;
-        for(portTickType xLastWakeTime;;
-            vTaskDelayUntil(&xLastWakeTime, state.callback_interval)) {
-                xLastWakeTime = xTaskGetTickCount();
+	portTickType wake_tick = 0;
+        for(;;) {
```

Comment:
More data. while at 1Hz, I rapidly pasted the following into interactive mode, with a carriage return in the clipboard, so it processed immediately after the paste. 

println('asdfasdf')


The commands were happening quickly with manual pastes, but if I held down the keyboard to have it repeat, then the commands started processing at 1Hz! interesting.